File: Python Word Examples/WatermarksApiDemo/setImageWatermarkInRequest.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.WatermarksApi(configuration)

file_name = 'Encrypt.docx'
input_image = 'D:/input/Logo.png'
dest_file_path = 'output/SetImageWatermarkInRequest_true.docx'
#Just moving "'password': '123'" if there is no password.
extra = {'folder':"input", 'scaling': 120,'washout': True, 'password': '123'}
result = api.set_image_watermark_in_request(file_name, input_image, dest_file_path, **extra)

File: JavaScript Word Examples/WatermarksApiDemo/setImageWatermarkInRequest.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.WatermarksApi(apiClient);

    var name = "Template.docx";
    var fs = require('fs');
    var file = fs.createReadStream("D://data//Logo.png");
    var destFilePath = "output/SetImageWatermarkInRequest_out.docx";
    var opts = {
        folder: 'input',
        storage: null,
        scaling: 120,
        washout: true,
        password: null
    };
    instance.setImageWatermarkInRequest(name, inputImage, destFilePath, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();